tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -137,7 +138,12 @@ Check HTTP Status Code
     [Return]  ${response.status_code}
 
 Load Expected Data Of RHODS Explore Section
-    ${apps_dict_obj}=  Load Json File  ${APPS_DICT_PATH}
+    ${version-check}=   Is RHODS Version Greater Or Equal Than  1.8.0
+    IF  ${version-check}==True
+        ${apps_dict_obj}=  Load Json File  ${APPS_DICT_PATH_LATEST}
+    ELSE
+        ${apps_dict_obj}=  Load Json File  ${APPS_DICT_PATH}
+    END
```

Comment:
Is the `LATEST` version of the JSON supposed to be used from 1.8 onwards? or did you mean 1.7?

Reply:
the intent is to use from 1.8 onwards